web-wallet/src/lib/components/Allocate/Allocate.svelte

```diff
@@ -21,9 +21,8 @@
     WizardStep,
   } from "$lib/dusk/components";
   import { GasFee, GasSettings, OperationResult } from "$lib/components";
-
-  /** @type {(to: string, amount: number, gasPrice: bigint, gasLimit: bigint) => Promise<string>} */
-  export let execute;
+  import { walletStore } from "$lib/stores";
```

Comment:
Per our conventions it's the `AllocateContract` that should access the store and supply the `execute` function to this component.

Reply:
Will open another issue to refactor this.